src/test/java/apicurito/tests/configuration/TestConfiguration.java

```diff
@@ -227,38 +227,38 @@ private Properties defaultValues() {
 
         if (props.getProperty(APICURITO_OPERATOR_CRD_URL) == null) {
             props.setProperty(APICURITO_OPERATOR_CRD_URL, String.format(
-                    "https://raw.githubusercontent.com/jboss-fuse/apicurio-operators/master/apicurito/deploy/crds/apicur_v1alpha1_apicurito_crd.yaml"));
-        }
-        if (props.getProperty(APICURITO_OPERATOR_DEPLOYMENT_URL) == null) {
-            props.setProperty(APICURITO_OPERATOR_DEPLOYMENT_URL,
-                    String.format("https://raw.githubusercontent.com/jboss-fuse/apicurio-operators/master/apicurito/deploy/operator.yaml"));
+                    "https://raw.githubusercontent.com/jboss-fuse/apicurio-operators/master/apicurito/config/crd/deprecated/apicur.io_apicuritoes.yaml"));
```

Comment:
We need the new CRD, not the deprecated one. 
Use this function to distinguish between 4.x and 3.11 OCPs: 
https://github.com/syndesisio/syndesis-qe/blob/461bf7b5dd3f5a2e3f12b33bec698ad2699c3986/utilities/src/main/java/io/syndesis/qe/utils/OpenShiftUtils.java#L420